### PR TITLE
Change machine to larger in release job template

### PR DESCRIPTION
**Pull request by konrad-sol**
We can switch to larger m-2 machine and check how it will impact performance and our costs in github

### CODE REVIEW

File: policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{ define "goreleaser" }}
   goreleaser:
     name: '{{`${{ matrix.golang_cross }}`}}'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-m-2
```
Comment (alephnull) [inline policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl:4]: We have two larger runners. I think we can do builds on one and tests on the other. ubuntu-latest-m-2 is the larger one. I forgot the name of the other one, @buger could you take a look at the Github org settings and tell us?

Comment (buger) [reply policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl:4]: This ones? 
image

Comment (alephnull) [reply policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl:4]: Yes, thank you.

**Review by alephnull — APPROVED** [review]
Thank you for this!

File: policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl (hunk 1 above)
Comment (konrad-sol) [reply policy/templates/releng/.github/workflows/release.yml.d/goreleaser.gotmpl]: I will check the performance on m-2 and m machines and post the results

**Pull request merged**